modules/gitops-repo-server-properties.adoc

```diff
@@ -16,16 +16,18 @@ The following properties are available for configuring the repo server component
 |`serviceaccount` |`""` |The name of the `serviceaccount` to use with the repo-server pod.
 |`verifytls` |`false` |Defines whether to enforce strict TLS checking on all components when communicating with repo server.
```

Comment:
```suggestion
|`verifytls` |`false` |Defines whether to enforce strict TLS checking on all components when communicating with the repo server.
```
Should "repo server" be "repository server"?

Reply:
No, it should be repo server.